source/blog/2018-08-31-the-ember-times-issue-62.md

```diff
@@ -41,8 +41,47 @@ Read more in the [rendered pull request on GitHub](https://github.com/emberjs/rf
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Module Unification with Ember Addons](https://github.com/emberjs/rfcs/pull/367) 🎁
+[Module Unification Packages](https://github.com/emberjs/rfcs/pull/367) is a new RFC, created by [@mixonic](https://github.com/mixonic), that sets out to describe how packages, **Ember Addons**, work and how apps and addons will migrate to the new [Module Unification](https://github.com/emberjs/rfcs/blob/master/text/0143-module-unification.md) structure from the classic structure. This RFC iterates on and is set to replace another RFC called [Module Unification Namespaces](https://github.com/emberjs/rfcs/pull/309) which had some syntax, like the `::` syntax, that proved problematic.
```

Comment:
> that sets out to describe how packages, **Ember Addons**

I suggest sticking with "describe how Ember apps and addons will migrate to the new...". The terminology of "package" is confusing if this is someones first introduction to the RFC. Focusing on "this is how addons and apps work under MU" is a good high level message.

Reply:
Ok makes sense. Thanks